src/Model/Table/RequestsTable.php

```diff
@@ -108,5 +108,10 @@ public function gc()
 
         $statement = $query->execute();
         $statement->closeCursor();
+
+        $conn = $this->getConnection();
+        if (is_a($conn->getDriver(), 'Cake\Database\Driver\Sqlite')) {
```

Comment:
We generally use `instanceof` instead of `is_a`

Reply:
ok, I changed that!